parse/kuneiform/gen/kuneiform_lexer.go

```diff
@@ -0,0 +1,483 @@
+// Code generated from KuneiformLexer.g4 by ANTLR 4.13.1. DO NOT EDIT.
```

Comment:
Was a git rm from kuneiform/gen/kuneiform_lexer.go?

Reply:
Oh, the diff must have exceeded it's threshold for matching.  All good.

Reply:
Ah I was not actually sure what was going on there